source/system/Mutex.ooc

```diff
@@ -56,8 +56,6 @@ MutexGlobal: class extends Mutex {
 	lock: override func { This _globalMutex lock() }
 	unlock: override func { This _globalMutex unlock() }
 	free: static func ~all { This _globalMutex free() }
-
-	GlobalCleanup register(|| This free~all())
```

Comment:
It's actually a different bug we're talking about. :)

But you are correct, this was run once for each class instance, which wasn't what we wanted either. This line was temporarily here because putting it in global scope in `Mutex.ooc` caused the line not to be executed at all. C code looks fine and there's no error, it just didn't do anything. I'm not sure if this is an ooc bug or how to make a good example.